frontend/pages/quotes/[pk].tsx

```diff
@@ -15,7 +15,9 @@ interface QuoteProps {
  */
 const Quote: FC<QuoteProps> = ({ quote }: QuoteProps) => {
   return (
-    <Layout title={""}>
+    // TODO: add attribute to Quotes for title text (not html),
```

Comment:
@mattfredericksen, yeah... It'd be nice if we could query quotes by slug, and then the slug base could be used for this title attribute...
Maybe we could add a field named `key` (which could contain some user-added string identifier, e.g.,  "what is freedom to the slave") to the Quote model and update Quote's `get_slug` method to return the slugified key.